components/autocomplete.js

```diff
@@ -72,12 +72,13 @@ const DropdownWithModal = ({ dataset, onChangeText, placeholder, setSelectedItem
       <View style={[styles.container, {justifyContent: 'space-between'}]}>
         <TextInput
           ref={inputRef}
-          style={[styles.input]}
+          style={textEditable ? [styles.input] : [styles.blockedInput]}
```

Comment:
i think they should still be able to type but their options are just limited, this looks like its blocking the user from typing ?

Reply:
Since we're switching the structure input to a Picker, I don't think this really matters anymore. I could remove this change if needed

Reply:
yes plz